submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionApiTest.kt

```diff
@@ -190,6 +192,38 @@ class SubmissionApiTest(
         assertThat(File("$ftpPath/${submitted.relPath}/Files/folder1")).isEmptyDirectory()
     }
 
+    @Nested
+    @SpringBootTest(webEnvironment = RANDOM_PORT, properties = ["app.subBasePath=subRelPath"])
+    inner class SubmitWhenSubmissionRelPath(@LocalServerPort val serverPort: Int) {
```

Comment:
`SubmitWebBasePath` this class is not related to relpath

Reply:
agree